static/refcache.json

```diff
@@ -123,9 +123,9 @@
     "StatusCode": 200,
     "LastSeen": "2023-02-15T20:41:36.268773-05:00"
   },
-  "https://cdn.jsdelivr.net/npm/mermaid@9.1.3/dist/mermaid.min.js": {
+  "https://cdn.jsdelivr.net/npm/mermaid@9.2.2/dist/mermaid.min.js": {
```

Comment:
Latest version in the v9 series is 9.4.0.

Meanwhile mermaid 10.0 was released (which is [ESM only](https://github.com/mermaid-js/mermaid/releases/tag/v10.0.0)!)

Reply:
Cool, and good to know. The version shown here matches the Docsy version at v0.6.0.